src/app/tabs/tabs.component.ts

```diff
@@ -54,6 +54,13 @@ export class TabsComponent extends RouterEnter {
         });
       });
     }
+    if (!this.utils.isMobile()) {
+      this.utils.getEvent('chat-bubble-update').subscribe(event => {
+        this.tabsService.getNoOfChats().subscribe(noOfChats => {
```

Comment:
no need to call api again to get the number of chats, it can be calculated here

Reply:
if we `event.readcount` to calculate. it will be the wrong count. because `readcount` is how many message read it does not always equal to unread message count it can be more than that. as `readcount` we got every message mark as read message-id Array length, we send every id to mark as read, even it's not an unread message. if we call API as we doing in other events we can get correct unread message count set it to tab.